public/kubernetes-guides/advanced-guides/hpa.mdx

```diff
@@ -0,0 +1,145 @@
+---
+title: "Horizontal Pod Autoscaling"
+---
+
+import { version } from '/snippets/custom-variables.mdx';
+
+The Horizontal Pod Autoscaler (HPA) automatically scales Kubernetes workloads based on resource utilization or other defined metrics.
+
+In Talos, HPA relies on the [Metrics Server ](../monitoring-and-observability/deploy-metrics-server)to collect data from the kubelets in your cluster. It then uses these metrics to decide when and how to scale your workloads.
+
+This guide walks you through how to use HPA to scale workloads on a Talos cluster.
+
+## Before you begin
+
+You’ll need the following to use HPA in your Talos cluster:
+
+* **A running Talos cluster**: If you don’t have one yet, see the <a href={`../../talos/${version}/getting-started/getting-started`}>Getting Started</a> or <a href={`../../talos/${version}/getting-started/prodnotes`}>Production Cluster</a> guides to create a cluster.
```

Comment:
do you make this ${version} so the same doc can be copied to multiple versions?

Reply:
Usually we say people need a "Kubernetes cluster on Talos Linux" instead of a "Talos cluster"